tests/networking/helper/helper.go

```diff
@@ -137,11 +137,13 @@ func GetClusterMultusInterfaces() ([]string, error) {
 	var nodesInterfacesList [][]string
 
 	for _, runningPod := range podsList.Items {
-		nodeInterfaces, err := getInterfacesList(runningPod)
-		if err != nil {
-			return nil, err
+		if !strings.Contains(runningPod.Spec.NodeName, "master") {
```

Comment:
This looks like a weak way to check whether a node is master. I recommend checking for the kubernetes standard node role label (node-role.kubernetes.io/master) instead. This can be done in a further PR, though.

Reply:
Agreed, thanks